### PR TITLE
 Make Currency Format Precision Consistent in Email Templates

### PR DESCRIPTION
**Pull request by SudharakaP**
Not sure if this is useful or not, but I thought I'll make the currency format precision consistent across all the email templates rather than just fixing the pending financial contribution template only. 😄 

Resolve https://github.com/opencollective/opencollective/issues/4713

### CODE REVIEW

**Comment by znarf** [comment]
Why not make precision=2 the default in the `currency` helper?

**Comment by SudharakaP** [comment]
> Why not make precision=2 the default in the `currency` helper?

That makes more sense. I've gone ahead and made the change so that in the email templates as well as the `formatCurrency` helpers the default precision is set to 2 decimals.

File: server/lib/handlebars.js [hunk 1]
```diff
@@ -125,7 +125,7 @@ handlebars.registerHelper('currency', (value, props) => {
     return value.toLocaleString(locale, {
       style: 'currency',
       currency,
-      minimumFractionDigits: precision || 0,
+      minimumFractionDigits: precision || 2,
```
Comment (Betree) [inline server/lib/handlebars.js:128]: Let's not impact the currencies without decimals

```suggestion
      minimumFractionDigits: precision || getDefaultCurrencyPrecision(currency),
```

Comment (SudharakaP) [reply server/lib/handlebars.js]: Thanks for pointing it out. Just corrected it. 👍🏽

**Review by Betree — APPROVED** [review]

**Pull request merged**